test/e2e/e2e-kind.sh

```diff
@@ -13,7 +13,7 @@ sudo mv kubectl /usr/local/bin/
 
 echo ">>> Building sigs.k8s.io/kind"
 cd $HOME
-GO111MODULE="on" go get -u sigs.k8s.io/kind@master
```

Comment:
`0.2.1` is from before they switched to Go modules iirc.

Reply:
Ahh, true. Thanks, I just amended the commit.